### ISSUE TITLE
this repo vs official integration

### ISSUE THREAD

**Issue by urielz** [Danielhiversen/home_assistant_airthings_cloud#36]
My understanding is that this add-on was recently merged into Home Assistant Core. Any advice on how to migrate to the official integration? I've been using this repo via HACS for a while. I don't know whether I would loose data if I delete this and install the official integration... ? Thanks for any help!

**Comment by iticory** [comment]
Oooh, I didn't know there was now an official [Airthings cloud integration](https://www.home-assistant.io/integrations/airthings/).  I see it even uses API keys instead of account passwords; nice!

I just completely removed my airthings config and this custom component from HACS.  After restarting and setting up the official integration, I was pleased to see that the history remained intact for the entities that had the same entity ID.  For the rest, I was able to change the entity ID to what the old integration used and it worked like a charm. ;)

It seems this repository hasn't been touched since the new one was released, so it may not be maintained anymore. 🤷
